src/view/plandetail/PlaceInfoTab.tsx

```diff
@@ -155,6 +170,10 @@ const TabPanelInformation = ({
                         label={`${priceRange.min}~\n${priceRange.max} 円`}
                     />
                 )}
+                <InformationTag
+                    icon={FaRegStar}
+                    label={averageRating.toFixed(1)}
+                />
```

Comment:
あと、ここがレビューが無いと表示できないから、条件文が必要かも
```suggestion
               {!isGooglePlaceReviewEmpty &&  <InformationTag
                    icon={FaRegStar}
                    label={averageRating.toFixed(1)}
                />}
```